modules/bitgo/src/v2/coins/sol.ts

```diff
@@ -283,10 +297,13 @@ export class Sol extends BaseCoin {
     ];
 
     const outputs: TransactionOutput[] = solTransaction.outputs.map((output) => {
-      return {
-        address: output.address,
-        amount: output.amount,
-      };
+      return JSON.parse(
```

Comment:
why?

Reply:
this one is because tokenName can be undefined so I want a way to not include the tokenName field when it's undefined.

Reply:
Object de-referencing is your friend.

```
const outputs: TransactionOutput[] = solTransaction.outputs.map(( { address, amount, tokenName }) => ({ address, amount, tokenName });

Reply:
Just tested and for this one, `tokenName` in `solTransaction.outputs` can exist with value undefined, so when its de-referenced, it stays while keeping `tokenName` undefined instead of removing it
(it cause failing in unit-test for native coin)

Reply:
ok. It could be that the unit test can just be updated, but if you really dont' want to return an object with `tokenName: undefined` then you probably just want to conditionally assign in a separate step.

```
solTransaction.outputs.map(( { address, amount, tokenName }) => {
  const output = { address, amount };
  if (tokenName) {
    output.tokenName = tokenName;
  }
  return output;
});

Reply:
Ohh okk, I'll just keep tokenName undefined to not showed then